addons/repair/models/stock_lot.py

```diff
@@ -7,23 +7,32 @@
 class StockLot(models.Model):
     _inherit = 'stock.lot'
 
-    repair_order_ids = fields.Many2many('repair.order', string="Repair Orders", compute="_compute_repair_order_ids")
-    repair_order_count = fields.Integer('Repair order count', compute="_compute_repair_order_ids")
+    repair_line_ids = fields.Many2many('repair.order', string="Repair Orders", compute="_compute_repair_line_ids")
+    repair_part_count = fields.Integer('Repair part count', compute="_compute_repair_line_ids")
+    in_repair_count = fields.Integer('In repair count', compute="_compute_in_repair_count")
     repaired_count = fields.Integer('Repaired count', compute='_compute_repaired_count')
 
     @api.depends('name')
-    def _compute_repair_order_ids(self):
+    def _compute_repair_line_ids(self):
         repair_orders = defaultdict(lambda: self.env['repair.order'])
-        for repair_line in self.env['stock.move'].search([('repair_id', '!=', False), ('move_line_ids.lot_id', 'in', self.ids), ('state', '=', 'done')]):
+        for repair_line in self.env['stock.move'].search([('repair_id', '!=', False), ('repair_line_type', '!=', False), ('move_line_ids.lot_id', 'in', self.ids), ('state', '=', 'done')]):
```

Comment:
More lines but better readability
```suggestion
        repair_moves = self.env['stock.move'].search([
            ('repair_id', '!=', False),
            ('repair_line_type', '!=', False),
            ('move_line_ids.lot_id', 'in', self.ids),
            ('state', '=', 'done')])
        for repair_line in repair_moves:
```